### PR TITLE
temporal 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by feedmeapples**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/temporal.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  sha256 "b8f83e07f47994482b73e3288083f38635aaeb77de673920041307254363f5be"
+  license "MIT"
+
+  bottle do
+    sha256 cellar: :any_skip_relocation, arm64_big_sur: "3f7c7bc02f74eb2a1a42c7fe9b152b512090acf29cd999da5c4d314712bb5d00"
+    sha256 cellar: :any_skip_relocation, big_sur:       "bc9baee6ccabcf595d07a207afc60eda1cb56d6eeed4efc46b4880143a504f7e"
+    sha256 cellar: :any_skip_relocation, catalina:      "2e5e643ffd01f0fceb0259a1e1dc8fa3d48e44bdbd9ecb1591efd7ddc181d3e7"
+    sha256 cellar: :any_skip_relocation, mojave:        "ebf347dc25bfb6adf5700d96078d4ef273cb36a522bb439ff45d8dc01e3193b2"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:  "a2be65b489682c2ab5d5677c9732e8af4f17038c2854a21cf77a1b19d034604c"
+  end
```
Comment (chenrui333) [inline Formula/temporal.rb:14]:
```suggestion
```

File: Formula/temporal.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/temporal/main.go"
```
Comment (chenrui333) [inline Formula/temporal.rb:19]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/temporal"
```

File: Formula/temporal.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+class Temporal < Formula
+  desc "Command-line interface for running and interacting with Temporal Server and UI"
+  homepage "https://temporal.io/"
+  url "https://github.com/temporalio/cli/archive/v0.2.0.tar.gz"
+  sha256 "b8f83e07f47994482b73e3288083f38635aaeb77de673920041307254363f5be"
+  license "MIT"
```
Comment (chenrui333) [inline Formula/temporal.rb:6]:
```suggestion
  license "MIT"
  head "https://github.com/temporalio/cli.git", branch: "main"
```

File: Formula/temporal.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  head "https://github.com/temporalio/cli.git", branch: "main"
+
+
```
Comment (feedmeapples) [inline Formula/temporal.rb:9]:
```suggestion
```

**Comment by feedmeapples** [comment]
ty for code review 🙏

File: Formula/temporal.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/temporal"
+  end
+
+  test do
+    run_output = shell_output("#{bin}/temporal --version 2>&1")
+    assert_match "temporal version", run_output
```
Comment (chenrui333) [inline Formula/temporal.rb:17]:
```suggestion
    run_output = shell_output("#{bin}/temporal --version")
    assert_match "temporal version #{version}", run_output
```

Comment (chenrui333) [reply Formula/temporal.rb:17]: One last nitpick comment

**Review by dawidd6 — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@feedmeapples, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**